packages/api/src/UserModule/UserModule.service.ts

```diff
@@ -88,13 +93,16 @@ export class UserModuleService {
     um.completedAt = new Date();
     await um.save();
 
+    await this.userService.incrementStreak(userId);
```

Comment:
So streak is incremented or reset when user begins a module?
I think it should be when a module is completed, e.g. increment streak by 1 after completing a module.

Reply:
Hey @shaneSheridan, you're right. There are two different implementations here: 1) when a streak has not been completed within the last 24 hours and a user logs on, for instance, and his or her user information is fetched we will reset the streak or not; and 2) when a user completes a module we will increment the streak. We will only increment  a user streak when a module is completed.

Reply:
Ok, so it will only be _incremented_ when a module is _completed_.